packages/workers-shared/asset-worker/tests/rules-engine.test.ts

```diff
@@ -115,3 +119,83 @@ describe("replacer", () => {
 		);
 	});
 });
+
+describe("static routing rules", () => {
```

Comment:
If you wanted a bunch of pre-built tests, you could grab these: https://cflare.co/url-glob-test. I think that's what Pages' stuff actually runs against today anyway.

Reply:
nice - not able to steal all of them due to the fact that the matcher only matches on Requests, but i stole a big chunk